api/v1alpha1/types.go

```diff
@@ -78,8 +78,7 @@ type Schema struct {
 	Status runtime.RawExtension `json:"status,omitempty"`
 	// Validation is a list of validation rules that are applied to the
 	// resourcegraphdefinition.
-	// Not implemented yet.
-	Validation []string `json:"validation,omitempty"`
+	Validation []Validation `json:"validation,omitempty"`
```

Comment:
Even thought this is technically a breaking change, this field was never documented nor used for anything in the past. So i'm ok with making this change to move fast.

thoughts @matthchr @barney-s ?

Reply:
Agreed